yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -10,29 +10,39 @@ import { makeAppendOnlyTreeSnapshot, makeHeader } from './l2_block_code_to_purge
  * The data that makes up the rollup proof, with encoder decoder functions.
  */
 export class L2Block {
+  #l1BlockNumber?: bigint;
+
   constructor(
     /** Snapshot of archive tree after the block is applied. */
     public archive: AppendOnlyTreeSnapshot,
     /** L2 block header. */
     public header: Header,
     /** L2 block body. */
     public body: Body,
-  ) {}
+    /** Associated L1 block num */
+    l1BlockNumber?: bigint,
+  ) {
+    this.#l1BlockNumber = l1BlockNumber;
+  }
 
   /**
    * Constructs a new instance from named fields.
    * @param fields - Fields to pass to the constructor.
    * @param blockHash - Hash of the block.
+   * @param l1BlockNumber - The block number of the L1 block that contains this L2 block.
```

Comment:
Reverting this accidental change https://github.com/AztecProtocol/aztec-packages/pull/5371.